src/mpi/romio/adio/common/ad_close.c

```diff
@@ -61,7 +61,8 @@ void ADIO_Close(ADIO_File fd, int *error_code)
     }
 
     if (fd->hints) ADIOI_Free(fd->hints->ranklist);
-    if (fd->hints) ADIOI_Free(fd->hints->cb_config_list);
+    if (fd->hints)
```

Comment:
The more general situation is BGQ does NOT utilize the cb_config_list hint because it has a hardware-specific mechanism for selecting the aggregators with its own environment variables, in theory this could apply to other architectures, hence the change in the common code